tests/bluetooth/tester/src/btp/btp.h

```diff
@@ -38,67 +38,69 @@
 #include "btp_tbs.h"
 #include "btp_tmap.h"
 #include "btp_ots.h"
+#include "btp_bas.h"
 
-#define BTP_MTU 1024
+#define BTP_MTU           1024
 #define BTP_DATA_MAX_SIZE (BTP_MTU - sizeof(struct btp_hdr))
 
-#define BTP_INDEX_NONE		0xff
-#define BTP_INDEX		0x00
+#define BTP_INDEX_NONE 0xff
+#define BTP_INDEX      0x00
```

Comment:
Suggest to undo these changes. Have you set your formatter to format all lines, and not just changed lines?

Reply:
Applies to several other lines in this file

Reply:
I didn't set it myself, so perhaps that's the default formatting option?

Anyways, will do, thanks for the suggestion :)

Reply:
> I didn't set it myself, so perhaps that's the default formatting option?

You may have misconfigured your IDE to format lines not changes by you - Most IDE can be configured to only format using e.g. clang-format on lines you've manually modified.